naloge/dectree3.tex

```diff
@@ -11,6 +11,46 @@
 \podnaslov{Odločitveno drevo}
 \end{slika}
 \end{vprasanje}
+
 \begin{odgovor}
+Izračunajmo najprej pričakovane vrednosti v vozliščih $C$, $F$ in $G$
+odločitvenega drevesa s slike~\fig.
+\begin{alignat*}{2}
+C &= 0.5 \cdot (40 p + 20) &&= 20p + 10 \\
+F &= 50 \cdot p - 10 \cdot (1-p) &&= 60 p - 10 \\
+G &= - 30 \cdot p + 30 \cdot (1-p) &&=  -60p + 30
+\end{alignat*}
+
+Obravnavajmo najprej odločitev v vozlišču $D$.
+Za pot k vozlišču $F$ se odločimo,
+če velja $60p - 10 \ge -5$ oziroma $p \ge 1/12$.
+
+Obravnavajmo sedaj še odločitev v vozlišču $E$
+-- za pot k vozlišču $G$ se odločimo,
+če velja $-60p + 30 \ge -5$ oziroma $p \le 7/12$.
+
+Pričakovana vrednost v vozlišču $B$ bo sledeča.
+\begin{align*}
+p &< {1 \over 12}: & 0.9 \cdot (-5) + (-60p + 30) \cdot 0.1 &= -1.5 - 6p \\
+\frac{1}{12} \leq p &\le \frac{7}{12}: &
+0.9 \cdot (60p - 10) + 0.1 \cdot (-60p + 30) &= -6 + 48p \\
+p &> \frac{7}{12}: & 0.9 \cdot (60p - 10) + 0.1 \cdot (-5) &= 9.5 + 54 p
+\end{align*}
+
+Obravnavajmo sedaj še odločitev v vozlišču $A$.
+\begin{align*}
+0 \leq p &< \frac{1}{12}: & \max \{ -1.5 - 6p, 20p + 10 \} &= 20p + 10 \\
+\frac{1}{12} \leq p &\le \frac{7}{12} : & \max \{ -6 + 48p, 20p + 10 \} \\
+p &> \frac{7}{12} : & \max \{ 9.5 + 54p, 20p + 10 \}
```

Comment:
Ta dva primera bo potrebno še razdelati v odvisnosti od *p*. Tukaj lahko to prikažeš še z `\begin{cases} ... \end{cases}`, končno odločitev pa prikažeš na drevesu na povezavah iz *A*.